src/plugins/intel_cpu/src/nodes/mkldnn_reduce_node.cpp

```diff
@@ -70,6 +71,40 @@ using namespace Xbyak;
 #define GET_PTR_NCD_BASE_PTR_N_BLK const uint8_t    *in_ptr_ncd    = in_ptr_n     + src_data_size * (icb * ID + id) * IH * IW * blk_size; \
                                          uint8_t    *out_ptr_ncd   = out_ptr_n    + dst_data_size * (ocb * OD + od) * OH * OW * blk_size;
 
+namespace {
+struct ReduceKey {
+    ReduceLayoutType layout;
+    Algorithm reduce_mode;
+    impl_desc_type implType;
```

Comment:
I don't agree that implType should be part of the key. Its choice depends only on existing HW capabilities which always const parameter during cache life frame.

Reply:
You are right. Fixed. Thanks Dmitry!